docs/source/cmd_new.rst

```diff
@@ -8,6 +8,31 @@ onyo new
    :path: new
 
 
+Reserved keys
+*************
+
+Onyo reserves some keys which have a special function when used while creating
+new assets, either with ``onyo new --keys`` or in a tsv table with
+``onyo new --tsv``.
+They can be used multiple times with different values to create multiple
+new assets at once with a different value for each asset.
+
+**directory**
+
+    The ``directory`` key is an alternative to ``onyo new --path`` to specify
+    the location in which to create new assets.
+
+**is_asset_directory**
+
+    Can be set to create new assets as an asset directory instead of an asset
```

Comment:
Are "asset directories" explained as a concept elsewhere in the docs yet? If so, it would be good to link "asset directory" to it.

Reply:
Thanks for your quick feedback! Nope, it isn't, but yes it should. 

It should also be stated that `is_asset_directory` can just be True or False. And this still needs to be verified/tested by me.

If this does not yet work, I might remove the description of the parameter as a whole from the "reserved keys" section and just describe what is tested and functions.

Reply:
It does not work for me, so I while delete the description of it until I now how and that it works.